help_data.yaml

```diff
@@ -76,3 +66,18 @@ livepatch:
       livepatches cannot be enabled on fips-enabled systems. You can find out
       more about Ubuntu Kernel Livepatch service at
       https://ubuntu.com/security/livepatch
+
+ros:
+  help: |
+    ros provides access to a private PPA which includes available high
+    and critical CVE fixes for Ubuntu Robot Operating System (ROS) packages
+    between the end of the standard Ubuntu LTS security maintenance and its
+    end of life. For full access to ROS ESM and security fixes, both
+    esm-infra and esm-apps services will also be enabled.
+    You can find out more about the esm service at
+    https://ubuntu.com/robotics/ros-esm
+
+ros-updates:
+  help: |
+    ros-updates provides the same security patches as ros and also includes
+    additional bugfixes.
```

Comment:
@artivis and @GAN-Canonical we could also use your input on this help text for the two services